pkg/operator/statefulset.go

```diff
@@ -28,6 +28,7 @@ const configReloaderPort = 8080
 // container. No shard environment variable will be passed to the reloader
 // container if `-1` is passed to the shards parameter.
 func CreateConfigReloader(
```

Comment:
It might be more readable to change the way we pass options to `CreateConfigReloader()` and use [functional options](https://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis) instead:
* more explicit than passing a zero  watch interval to enable the "run once" mode 
* some options don't make sense in this case (e.g. reload URL and ports).

Instead we could write something like:

```
initReloaderContainer := operator.CreateConfigReloader(
    "init-config-reloader",
    operator.ReloaderResources(c.ReloaderConfig),
    operator.ReloaderRunOnce(),
)

...

reloaderContainer := operator.CreateConfigReloader(
    "config-reloader",
    operator.ReloaderResources(c.ReloaderConfig),
    operator.ReloaderURL(url.URL{...}),
    ...,
)


```